less/c2corg_ui.less

```diff
@@ -30,6 +30,7 @@
 @import "icon-date.less";
 @import "context-help.less";
```

Comment:
CSS is not really my part so I trust your changes (but I see extra blank lines :P nevermind!)
At least the rendering is OK for various screen sizes with FF, Chrome and Safari (I don't have Edge).